app/javascript/components/layout/Header.js

```diff
@@ -52,7 +56,7 @@ class Header extends React.Component {
               <Form inline className="ml-auto">
                 <Navbar.Text className="text-white py-0 px-3">
                   <i className="fas fa-user fa-fw mr-2"></i>
-                  {this.props.current_user?.email}
+                  {this.props.current_user?.email} (<span className="capitalize">{this.props.current_user?.role.split('_').join(' ')}</span>)
```

Comment:
Not a huge thing, but why not just use `_.capitalize` from lodash here instead of adding a new class. I dont have strong feelings one way or the other, but it does reduce the number of lines.

Reply:
Haha I made the same comment above. It wouldn't actually reduce the number of lines though because lodash would need to be imported into this file.

Reply:
I didn't use it because I forgot it existed and defaulted to StackOverflow. I've swapped it over to lodash now, since it's a way cleaner method.